test/Misc/testFloat.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %swift -parse-stdlib -emit-assembly %s -o - | %FileCheck %s --check-prefix=CHECK-%target-cpu
+
+// test converting floating point to unsigned integer
+import Swift
+let f = Float(-123.0)
```

Comment:
Nitpick: `-123.0 as Float` is slightly better Swift. This way constructs a Double first and then truncates it.